Kubernetes 1.25 support (#593)
longhorn/website: content/docs/1.4.0/snapshots-and-backups/csi-snapshot-support/enable-csi-snapshot-support.md

> For example, on a vanilla Kubernetes cluster, update the namespace from `default` to `kube-system` prior to issuing the `kubectl create` command.

Install the Snapshot CRDs:

> 1. Download the files from https://github.com/kubernetes-csi/external-snapshotter/tree/v5.0.1/client/config/crd

Reviewer: Will it be helpful to mention installing a compatible version of the built-in external-snapshotter CSI sidecar?

Author: external-snapshotter CSI sidecar is shipped by Longhorn. Could you explain a bit more? Thank you

Reviewer: Suggest explaining why the version mentioned is `v5.0.1` or a compatible one, this is because it's referenced from the version of the external-snappshotter used in Longhorn, right?

Author: Ah. I see. Thank you. Fixed

2. Run `kubectl create -f client/config/crd`.
3. Do this once per cluster.

Install the Common Snapshot Controller:
1. Download the files from https://github.com/kubernetes-csi/external-snapshotter/tree/v5.0.1/deploy/kubernetes/snapshot-controller
2. Update the namespace to an appropriate value for your environment (e.g. `kube-system`)
3. Run `kubectl create -f deploy/kubernetes/snapshot-controller`.
3. Do this once per cluster.
> **Note:** previously, the snapshot controller YAML files were deployed into the `default` namespace by default.
> The updated YAML files are being deployed into `kube-system` namespace by default.

> > Therefore, you might want to delete the previous snapshot controller in the `default` namespace to avoid having multiple snapshot controllers.

Reviewer 2: ```suggestion
> Therefore, we suggest deleting the previous snapshot controller in the `default` namespace to avoid having multiple snapshot controllers.
```

Author: Thank you. Fixed
